OptionsDialog.cs

```diff
@@ -303,7 +306,8 @@ void cmdOK_Click(object sender, EventArgs e)
 			int temp = 0;
 			for (int i = 0; i < 22; i++) temp += (chkWP[i].Checked ? 1 << i : 0);
 			_config.Waypoints = temp;
-			_config.MapOptions = (chkFG.Checked ? Settings.MapOpts.FGTags : Settings.MapOpts.None) | (chkTrace.Checked ? Settings.MapOpts.Traces : Settings.MapOpts.None);
+			_config.MapOptions = (_config.MapOptions ^ (_config.MapOptions & Settings.MapOpts.FGTags)) | (chkFG.Checked ? Settings.MapOpts.FGTags : Settings.MapOpts.None);
```

Comment:
We're re-initializing MapOptions here, so "turning off" the option isn't necessary

Reply:
May have jumped the gun on this comment, still digging into the map itself to see how the new options are managed

Reply:
The purpose of the change here was that it would erase the new flags, so now it specifically masks each of the flags it's checking.  Technically the FGTags and Traces checkboxes in Options are now obsolete, as MapForm saves those settings during form_FormClosing()
My reasoning here was for some of the new map flags to be persistent.  Mainly it was just for TraceSelected since that makes the trace lines behave similarly to AlliED, and if people are accustomed to that behavior they might want the change to stick.

Reply:
Yeah, I see that now.  Might pull it out of Options